front/app/utils/configs/votingMethodConfig/textNumberOfVotes.ts

```diff
@@ -35,11 +35,9 @@ const getTextNumberOfVotes = ({
     formatMessage,
   });
 
-  return formatMessage(messages.numberOfVotes, {
-    numberOfVotes: numberOfVotes.toLocaleString(),
-    voteTerm,
-    votesTerm,
-  });
+  return `${numberOfVotes.toLocaleString()} ${
```

Comment:
Why would you call toLocaleString on numberOfVotes?

Reply:
Do you want to show text like "zero votes", "one vote", etc.?

Reply:
I don't think it converts numbers to words, I believe it's just for edge cases like NaN and float numbers https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
I just copied it from another place where we use this `numberOfVotes`.